feature/experimental/policy/policy_vrf_selection/ate_tests/base_vrf_selection/base_vrf_selection_test.go

```diff
@@ -362,49 +390,58 @@ func TestVrfPolicy(t *testing.T) {
 	p1 := dut.Port(t, "port1")
 	p2 := dut.Port(t, "port2")
 
-	// Configure DUT interfaces and forwarding policy
+	// Configure DUT interfaces and forwarding policy.
 	configureDUT(t, dut, p1, p2)
+	configDefaultRoute(t, dut, ateDestIPv4VLAN20, defaultNHv4, ateDestIPv6, defaultNHv6)
+	configvrfRoute(t, dut, ateDestIPv4VLAN10, vrfNH)
 
-	// Configure ATE
+	// Configure ATE.
 	ate := ondatra.ATE(t, "ate")
-	allFlows := configureATE(t, ate)
+	configureATE(t, ate)
+
+	allFlows := createTrafficFlows(t, ate)
 
 	tcs := []struct {
-		desc      string
-		policy    string
-		passFlows []string
+		desc        string
+		policy      string
+		endpoints   []string
+		streamNames []string
```

Comment:
I think an earlier comment I made about the names here was unclear.  To mimic verifyTraffic, let's rename these fields:
  streams -> flows
  streamNames -> passFlows